core/src/main/java/org/elasticsearch/common/lucene/LoggerInfoStream.java

```diff
@@ -53,14 +46,11 @@ public boolean isEnabled(String component) {
     }
 
     private Logger getLogger(String component) {
-        if (component.equals("IFD")) {
-            return ifdLogger;
-        } else {
-            return logger;
-        }
+        return Loggers.getLogger(parentLogger, "." + component);
```

Comment:
I think this change inadvertently slowed down indexing: see annot AZ at https://benchmarks-old.elastic.co/index.html#indexing

The problem is (confusingly!) this private method is called for every log message that Lucene's `IndexWriter` makes.  Previously, we invoked `Loggers.getLogger` just twice on creating `LoggerInfoStream` but now we are calling it for every log message.